cypress/e2e/news/createReadDeleteSchoolNewsInAdvance.feature

```diff
@@ -18,6 +18,7 @@ Feature:  News - To read a news on the respective dashboards
         When I set news-visibility-start-date to '<news_day_from_today>' days at '<news_time>'
         Then I see time input field
         When I click on save button
+        Then I can read the news '<news_title>' with description '<news_description>' on news detail page
```

Comment:
instead of  'on news detail page' it should be--- >  'on the page News Detail'

Reply:
I don't think so as there are numerous places we have same pattern, search for `detail page`, `details page`, `overview page` etc in the repo. 

if the think will be as you suggested then we need another followup ticket which mentions all places as doing for some feature files will cause test failures.

Reply:
yes, this is kinda refactoriing or missed out in the previous PRs, it wont be a huge effort, just simple search & replace, and it will be replaced in all files.

Reply:
https://ticketsystem.dbildungscloud.de/browse/BC-9321